en/components/accordion.md

```diff
@@ -0,0 +1,59 @@
+
+## Accordion Demo
+
+<img class="responsive-img" src="../images/accordion_demo.png" srcset="../images/accordion_demo@2x.png 2x" />
+
+## Using the Accordion in Figma and Sketch
+
+The Accordion comes with 5 panels. Each panel consists of top and bottom margins, which can be turned on/off with the boolean operations. By going into the nested Expansiоn Panels, you can modify their contents. If you need to design an Accordion with more Expansion Panels, the easiest way to do it is by detaching the component. To detach the component, you need to place it in your workspace (Frame or Artboard). Right-clicking on it will open a contextual menu where `Detach Instance` (`Detach from Symbol` in Sketch) should be selected.
+
+After detaching the component, you will be able to add your additional Expansion Panels by duplicating any of the existing ones. In Figma, the size will be adjusted automatically thanks to the Auto Layout. In Sketch, you should manually adjust the additionally added Expansion Panels' position.
+
+## Using the Accordion in Adobe XD
+
+> [!NOTE]
+> WIP: Currently, we don't support the Accordion component in Adobe XD, but you can also combine multiple `Expansion Panels` into a similar Accordion component behavior. However, this will not generate appropriately in the App Builder.
+
+## Interaction States
+
+The Accordion doesn't have interaction states, but its panels have `Disabled` and `Expanded,` which can be toggled on and off for the selected Expansion Panel. Check [Expansion Panels](expansion-panel.md)
+
+<img class="responsive-img" src="../images/accordion_disabled-panel.png" srcset="../images/accordion_disabled-panel@2x.png 2x" />
+
+In Figma, the `Expanded` property gives you the ability to configure the states of the panel as per your requirement. Additionally, based on the panel hierarchy in the Layers panel and which panel is expanded, you can activate the top and bottom margins from boolean properties, adding more space to your designs. In Sketch, there is a predefined symbol, `Single Branch Expanded,` covering the expanded state of the panel and its margins.
+
+<img class="responsive-img" src="../images/accordion_expanded-panel.png" srcset="../images/accordion_expanded-panel@2x.png 2x" />
+
+## Styling
+
+The Accordion component offers styling flexibility, with various options for customization. You can choose the colors of both the Header's and Body's background, as well as the text color and weight. Additionally, you can customize the icon color and type.
```

Comment:
Maybe we can change Header's and Body's background to Headers' and Bodies' backgrounds, because there are a few headers and bodies, not single ones.